Jbarget/oxymore
# creates advertising page #7

### What changes have you made?

- created advertising page with images and placeholder text 
- image sizes reduced 

### Which issue(s) does this PR fix?

Fixes #7 

### Screenshots (if there are design changes)

<img width="1065" alt="Screenshot 2020-09-16 at 12 46 49" src="https://user-images.githubusercontent.com/53219789/93327495-b9e74100-f81a-11ea-921b-4f3669a571ab.png">


<img width="225" alt="Screenshot 2020-09-14 at 16 29 59" src="https://user-images.githubusercontent.com/53219789/93099541-67861300-f6a8-11ea-9f42-52e2bbbd5f4a.png">

### How to test
open Advertising page and check: 
- that the images appear side by side on desktop with text underneath
- that the images and text appear in a column on mobile
